webdriver_manager/driver_cache.py

```diff
@@ -57,7 +57,8 @@ def save_driver_to_cache(self, response, driver_name, version, os_type):
         file_path = os.path.join(driver_path, filename)
         write_file(response.content, file_path)
         files = self.__unpack(file_path)
-        return os.path.join(driver_path, files[0])
+        file_path = os.path.join(driver_path, files[0])
```

Comment:
no need to assign a variable here, just return

Reply:
fixed